constants/constants.go

```diff
@@ -103,6 +103,9 @@ var (
 
 	/*EthtoolFilters*/
 	ethtoolFilterRegex = `^[a-zA-Z0-9-:.-/\s/g]+$` // regex to validate ethtool filter commands.
+
```

Comment:
Shouldn't need any changes in this file at all
I think `git reset HEAD <file>` could be the quickest way to remove